packages/breadboard-ui/src/types/types.ts

```diff
@@ -94,3 +95,11 @@ export type OutputArgs = {
     schema?: Schema;
   } & Record<string, unknown>;
 };
+
+export type BreadboardElementError = InputErrorEvent; //more error types can be added for different components later on i.e., "& ElementErrorEvent & OtherElementErrorEvent" and so on
```

Comment:
I think we're super close - just this part seems a bit unusual to me. In the consuming code, if we're wanting to do a rename we could do:

```ts
import { type InputErrorEvent as BreadboardElementError} from '@google-labs/breadboard-ui';
```
In other words, I don't know that we need dedicated types here. The same goes for `BreadboardErrorHandler` (which I don't _think_ is needed now?), nor `BreadboardReactComponentProps`, which I wouldn't expect to live in these types, but rather in the code that is consuming Breadboard and wrapping it for use with React.

Reply:
Just to explain a bit more what I had in mind with the BreadboardElementError type, I'll give a bit more context:

See https://github.com/ExaDev-io/react-breadboard-components/blob/react-component-wrappers/src/lib/errors.ts

In the function at the above link, the idea was to be able to make the type of the "error" parameter more generic rather than have it as InputErrorEvent only, so that the handleError function could be used for other components that we might wrap in the future. 

However, I agree that these could instead live in the code that is consuming breadboard (in my case, in React). So, for this PR, I will remove them from the breadboard-ui types.